libs/styles/styles.css

```diff
@@ -218,6 +218,12 @@ picture.bg-img img {
   visibility: hidden;
 }
 
+/* merch */
+[data-wcs-osi],
+a[href^="/tools/ost"] {
```

Comment:
This shouldn't be needed.

Reply:
@auniverseaway this is to prevent merch links being displayed while waiting for merch block logic and styles.
Otherwise it gives a flicker effect. The unprocessed merch links should not be visible to the visitor.

Reply:
This is also needed to hide unresolved placeholders while merch.css is still being loaded.
Happened several times during development.

Reply:
shouldn't we in that case add a class for auto blocks sharing that requirement? @tuerkben @auniverseaway